sdcm/tester.py

```diff
@@ -1271,6 +1268,41 @@ def pre_create_alternator_tables(self):
                 self.alternator.update_table_ttl(node=node, table_name=alternator.consts.TABLE_NAME)
                 self.alternator.update_table_ttl(node=node, table_name=alternator.consts.NO_LWT_TABLE_NAME)
 
+    def pre_create_alternator_backuped_tables(self, **kwargs):
```

Comment:
I'd prefer not adding more to `tester.py` file.
I propose to add `node`, `params` and `alternator` parameters so this method could be moved to `sdcm.utils.alternator` module.
(same for methods/ctx mgr below)